setup.py

```diff
@@ -14,7 +14,8 @@
 
 
 install_requires = [
-    'jsonschema',
+    # TODO: unpin this
```

Comment:
Is there an issue or some indication in which we can explicitly verify compatibility so we can unpin this later?

Reply:
I added an issue to the comment. However, to be clear, `make test` currently fails with an import error without this max pin.